### DISCUSSION TITLE
Clarification on which files to include for Lab 1 Submission

### DISCUSSION THREAD

**Discussion by ElectricPyramid** [NUS-CG3207/labs#4] [Lab 1] [answered]
In the Lab 1 Manual, this is stated _"DO NOT zip up the complete project folders or individual files one by one. The files should be the exact same files that you used for the demo"._  I would like to clarify which files should be included if not the entire modified lab1 folder zipped? Should the needed files be copied into another folder (disregarding the original directory hierarchy) and then that folder should be zipped?

**Comment by NeilBaner** [comment, marked as answer]
> Should the needed files be copied into another folder (disregarding the original directory hierarchy) and then that folder should be zipped?

Yes, correct.
